sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncRepository.kt

```diff
@@ -25,18 +25,25 @@ import com.duckduckgo.sync.store.SyncStore
 import com.squareup.anvil.annotations.ContributesBinding
 import com.squareup.moshi.JsonAdapter
 import com.squareup.moshi.Moshi
-import javax.inject.Inject
+import javax.inject.*
+import kotlin.DeprecationLevel.WARNING
 import timber.log.Timber
 
 interface SyncRepository {
     fun createAccount(): Result<Boolean>
+
+    @Deprecated(message = "Method only used for testing purposes. Relies on a local stored recovery key.", level = DeprecationLevel.WARNING)
```

Comment:
can you make this internal then?

Reply:
I assume you mean `internal` visibility.

The repository itself is not exposed, so it's only used inside the :impl module. Adding `internal` will make no difference in this case.